spec/masamune/actions/elastic_mapreduce_spec.rb

```diff
@@ -78,7 +78,10 @@
       before do
         expect(instance).to_not receive(:elastic_mapreduce)
       end
-      it { expect { subject }.to_not raise_error }
+      it do
+        expect { subject }.to_not raise_error
+        expect(instance.configuration.elastic_mapreduce[:jobflow]).to eq('j-XYZ')
```

Comment:
ensure that `options[:jobflow]` is assigned to `configuration.elastic_mapreduce[:jobflow]`